osu.Game/Screens/Select/Details/UserRatings.cs

```diff
@@ -29,11 +29,17 @@ public BeatmapMetrics Metrics
                 if (value == metrics) return;
                 metrics = value;
 
-                var ratings = Metrics.Ratings.ToList();
-                negativeRatings.Text = ratings.GetRange(0, ratings.Count / 2 + 1).Sum().ToString();
-                positiveRatings.Text = ratings.GetRange(ratings.Count / 2 + 1, ratings.Count / 2).Sum().ToString();
-                ratingsBar.Length = (float)ratings.GetRange(0, ratings.Count / 2 + 1).Sum() / ratings.Sum();
-                graph.Values = Metrics.Ratings.Select(r => (float)r);
+                const int rating_range = 10;
+
+                var ratings = Metrics.Ratings.ToList().GetRange(1, rating_range); // adjust for API returning weird empty data at 0.
+
+                var negativeCount = ratings.GetRange(0, rating_range / 2).Sum();
+                var totalCount = ratings.Sum();
+
+                negativeRatings.Text = negativeCount.ToString();
+                positiveRatings.Text = (totalCount - negativeCount).ToString();
+                ratingsBar.Length = (float)negativeCount / totalCount;

```

Comment:
Can we maybe early return or set this to 0 if `totalCount` == 0? I.e. the map hasn't been rated yet?